src/hotspot/share/gc/parallel/mutableSpace.hpp

```diff
@@ -62,11 +62,13 @@ class MutableSpace: public CHeapObj<mtGC> {
 
   MutableSpaceMangler* mangler() { return _mangler; }
 
-  void numa_setup_pages(MemRegion mr, bool clear_space);
+  void numa_setup_pages(MemRegion mr, size_t page_size, bool clear_space);
 
   void set_last_setup_region(MemRegion mr) { _last_setup_region = mr;   }
   MemRegion last_setup_region() const      { return _last_setup_region; }
 
+  size_t used_page_size() const;
```

Comment:
I don't see any uses of this, which is fortunate since I don't see a definition either.

Reply:
Oh, that is a renmant of an earlier attempt, I'll remove it.